client/src/lib.rs

```diff
@@ -87,6 +91,9 @@ static LONG_POLL_METHOD_NAMES: [&str; 3] = [
 const LONG_POLL_TIMEOUT: Duration = Duration::from_secs(70);
 const OTHER_CALL_TIMEOUT: Duration = Duration::from_secs(30);
 
+/// Buffer size for the channel that listens to change events
+const DEFAULT_BUFFER_SIZE: usize = 1024;
```

Comment:
I suspect this can probably be a lot smaller (though I doubt it matters a whole lot either way).

Reply:
I just copied the constant used internally by tonic, frankly, no idea about the right size... But, yes, being a control channel, that looks large...